core/src/test/java/org/apache/spark/sql/delta/MergeIntoJavaSuite.java

```diff
@@ -42,6 +42,8 @@ public class MergeIntoJavaSuite implements Serializable {
     public void setUp() {
         spark = new TestSparkSession();
         tempPath = Utils.createTempDir(System.getProperty("java.io.tmpdir"), "spark").toString();
+        // disable the spark conf check
+        spark.sqlContext().conf().setConfString("spark.databricks.delta.requiredSparkConfsCheck.enabled", "false");
```

Comment:
Could you set the catalog rather than disabling the check?

Reply:
I have reverted the changes. My local upstream/master was didn't have the latest version at the time I made this change. I have now reverted the changes.